### ISSUE TITLE
Training and validation obj_loss with weird values

### ISSUE THREAD

**Issue by carlosmunozledesma** [ultralytics/yolov5#4923]
## ❔Question
I have a model pretrained (with yolov5s.pt) and now I'm training it with 1000 cars and plates. If I plot the obj_loss values from training and validation (from results.csv) I have a weird graph. Obj_loss training  is more than obj_loss validation. I don't know why. I think that it would have to be the other way around. This is my graphs:
![image](https://user-images.githubusercontent.com/21873207/134781194-f34ca436-c706-4439-b419-04cdbd0fef33.png)
![image](https://user-images.githubusercontent.com/21873207/134781229-000cbbe9-6e81-43dd-9a08-91f8296e6faf.png)

## Additional context
I have read this post [https://machinelearningmastery.com/learning-curves-for-diagnosing-machine-learning-model-performance/]
and this post says that it's because _"It may also be identified by a validation loss that is lower than the training loss. In this case, it indicates that the validation dataset may be easier for the model to predict than the training dataset."_ but if I change the images from train, valid and test folders I get the same result.

Thanks a lot.

**Comment by glenn-jocher** [comment]
@carlosmunozledesma yes this expected. Training data augmentation increases loss.

**Comment by carlosmunozledesma** [comment]
> @carlosmunozledesma yes this expected. Training data augmentation increases loss.
Thanks! Can I disable training data augmentation?

**Comment by glenn-jocher** [comment]
@carlosmunozledesma you can do anything you want. Hyperparameters set augmentation levels:
https://github.com/ultralytics/yolov5/blob/39c17ce0b922bbafcf9f8da64f286fef01040727/data/hyps/hyp.scratch.yaml#L22-L34

**Comment by Marks1995** [comment]
> hello world

**Comment by carlosmunozledesma** [comment]
@glenn-jocher Thanks!
